hooks/pre-commit

```diff
@@ -2,7 +2,7 @@
 
 set -e
 
-npm i --package-lock-only
+npm install
 npm run lint --fix
 npm run build
 git add src/
```

Comment:
This will likely lead to issues pertaining to things being unintentionally committed. This hook might need to be revisited to address its functionality could best fit in.

Reply:
True, this is meant to be a convenience hook and `git add src/` is meant to capture the changes to `npm run lint --fix`, but I see your point. I'll make a change here.